mods/rv/rules/ai-only-units.yaml

```diff
@@ -0,0 +1,229 @@
+### This file defines units can be only built by AI player, which
+### 1. will use diffirent set of logic compared to orignal units used by AI
+### 2. are pre-loaded transprot units with unload logic.
+
+fv-e1:
+	Inherits: fv
+	Buildable:
+		BuildPaletteOrder: 31
+		Prerequisites: ~prerequisite.aiplayer, ~techlevel.1
+	ProductionCostMultiplier:
+		Multiplier: 125
+	Cargo:
+		InitialUnits: e1
+
+fv-ggi:
+	Inherits: fv-e1
+	Buildable:
+		BuildPaletteOrder: 32
+		Prerequisites: ~prerequisite.aiplayer, ~techlevel.1
+	ProductionCostMultiplier:
+		Multiplier: 150
+	Cargo:
+		InitialUnits: ggi
+
+fv-gren:
+	Inherits: fv-e1
+	Buildable:
+		BuildPaletteOrder: 33
+		Prerequisites: ~prerequisite.aiplayer, gaairc, ~techlevel.2
+	ProductionCostMultiplier@1:
+		Multiplier: 150
+	Cargo:
+		InitialUnits: gren
+
+fv-cleg:
+	Inherits: fv-e1
+	Buildable:
+		BuildPaletteOrder: 33
+		Prerequisites: ~prerequisite.aiplayer, gatech, ~techlevel.4
+	ProductionCostMultiplier@1:
+		Multiplier: 150
+	Cargo:
+		InitialUnits: cleg
+	
+
```

Comment:
There is an extra whitespace and a tab here.

Reply:
fixed